.editorconfig

```diff
@@ -1,14 +1,901 @@
-# For ktlint configuration. Ref: https://ktlint.github.io/
-
-[*.{kt,kts}]
-# possible values: number (e.g. 2), "unset" (makes ktlint ignore indentation completely)
-indent_size=unset
-# true (recommended) / false
-insert_final_newline=true
-# possible values: number (e.g. 120) (package name, imports & comments are ignored), "off"
-# it's automatically set to 100 on `ktlint --android ...` (per Android Kotlin Style Guide)
-max_line_length=off
-
-# From https://github.com/pinterest/ktlint#custom-ktlint-specific-editorconfig-properties
-# default IntelliJ IDEA style, same as alphabetical, but with "java", "javax", "kotlin" and alias imports in the end of the imports list
-ij_kotlin_imports_layout=*,java.**,javax.**,kotlin.**,^
+[*]
```

Comment:
what was the command to generate this file OOI?

Reply:
![image](https://user-images.githubusercontent.com/20701752/163191634-b7299eb0-52ee-48c3-9989-0b1fe26fab03.png)

;)

Reply:
TIL, thanks!